README.md

```diff
@@ -2,4 +2,4 @@
 
 Many components and code of this repository are based on the [DigiWF project](https://github.com/it-at-m/digiwf-core) .
 
-![Platform Overview](./docs/imgs/miranum_platform.png)
+![Platform Overview](./docs/imgs/miranum-platform.png)
```

Comment:
The picture looks great. Well done 😃 

Why did you add the logos of keycloak, nginx, postgres, minio to the right side? I would place them outside of the platform. They are not really part of the platform and interchangeable.

Reply:
Well they're kinda part of the platform since its not working without the technologies.
It mainly depends on the customers infrastructure what needs to be added. But since the services are part of our docker-compose stack, I've added them inside the platform.

Reply:
I just added your point to the readme. Thanks for clarifying!